validation/validate.py

```diff
@@ -170,17 +170,80 @@ def _score_k_fold(thresholds: np.ndarray,
     return accuracy, recall, precision
 
 
-def _get_target_faces(embeddings1: List[List[float]],
-                      embeddings2: List[List[float]],
+def _get_target_faces(embeddings1: List[FaceVector],
+                      embeddings2: List[FaceVector],
                       distance_metric: DistanceMetric,
-                      is_match: bool) -> Tuple[List[float], List[float]]:
+                      is_match: bool) -> Tuple[FaceVector, FaceVector]:
     X, Y = zip(*[(emb1, emb2) for emb1 in embeddings1 for emb2 in embeddings2])
     distances = _distance_between_embeddings(X, Y, distance_metric)
     distance_criteria = min if is_match else max
     index, _ = distance_criteria(enumerate(distances), key=lambda x: x[1])
     return X[index], Y[index]
 
 
+def _get_container_metrics(face_vectors: List[List[FaceVector]]) -> Tuple[
+                                                                        int,
+                                                                        int,
+                                                                        float]:
+    num_expected = len(face_vectors)
+    num_missing = sum([1 for i in face_vectors if not i])
+    percentage_missing = 100 * (num_missing / num_expected)
+    return num_expected, num_missing, percentage_missing
+
+
+def _remove_empty_embeddings(args: Namespace,
```

Comment:
Why pass the args to check the boolean and short-circuit the function if it's false instead of not calling the function in the first place?